inscriptions/ISic003429.xml

```diff
@@ -61,9 +63,7 @@
                         <country>Italy</country>
                         <region>Sicily</region>
                         <settlement>Siracusa</settlement>
-                        <repository role="museum">Area archeologica della Neapolis, Orecchio di Dionisio e Teatro
-                            Greco
-                        </repository>
+                        <repository role="museum">Area archeologica della Neapolis, Orecchio di Dionisio e Teatro Greco</repository>
```

Comment:
Nb change @ role to "site" and provide ref="http://sicily.classics.ox.ac.uk/museum/160"